pkg/kube/cluster-init.sh

```diff
@@ -156,9 +156,7 @@ check_start_containerd() {
                 # This is very similar to what we do on kvm based eve to start container as a VM.
                 logmsg "Trying to install new external-boot-image"
                 # This import happens once per reboot
-                ctr -a /run/containerd-user/containerd.sock image import /etc/external-boot-image.tar docker.io/lfedge/eve-external-boot-image:latest
-                res=$?
-                if [ $res -eq 0 ]; then
+                if ctr -a /run/containerd-user/containerd.sock image import /etc/external-boot-image.tar docker.io/lfedge/eve-external-boot-image:latest; then
```

Comment:
This is fine for now, but I suspect that at some point in time we want to run a specific semver or hash and not :latest to avoid changes in production just because a device rebooted and picked up a new latest.

Reply:
Yes that's a good point, I feel like this may be a good thing to fix now before it is forgotten.  I can go ahead and set a specific version for now.

Reply:
In the interest of time, I'll get back to setting a version here in a followup PR.